tests/swig.at

```diff
@@ -34,10 +34,9 @@ m4_define([TEST_EVAL_SWIG],[
   m4_define([FACTS],[TESTDIR/facts])
   m4_define([CURRENTDIR],[`pwd`])
 
-  for file in `ls TESTDIR | grep -v .csv$`
-  do
-    cp -R "TESTDIR/$file" .
-  done
+  AS_IF([test -d FACTS],[cp -R FACTS .],[])

```

Comment:
Instead of copying the fact directory, can you just run souffle with `-F FACTS`? Similarly, can you specify the path to the datalog file rather than copying it (similar to TEST_EVAL in testsuite.at)?

If for some reason you do need to copy it, you could script it yourself a little more concisely without the AS_IF:
`test -d FACTS && cp -R FACTS .`

Reply:
Ok, so, after discussion, we have a better idea. Take a look at [this line in interface.at](https://github.com/souffle-lang/souffle/blob/aea32fa8985b408943a276373905535751ca202b/tests/interface.at#L21) and see how it passes the facts directory to [this line of the `load_print` test](https://github.com/souffle-lang/souffle/blob/aea32fa8985b408943a276373905535751ca202b/tests/interface/load_print/driver.cpp#L42). If you can use this technique, you can avoid copying the facts directory entirely.

Reply:
fixed